src/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ add_dependencies(chargefw2 ${METHODS})
 
 target_link_libraries(chargefw2 structures ${COMMON_LIBS} formats utility lhs Boost::program_options dl Boost::filesystem fmt::fmt)
 
+
```

Comment:
That's unnecessary.

Reply:
I am assuming you are talking about line 35?

Reply:
Yep.